packages/cfd/src/Constants/routes-config.tsx

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
-import CFD from '../Containers';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import CFD from '../Containers';
+import { TCFDDashboardProps } from '../Containers/cfd-dashboard';
+import { TRouteConfig } from '../types/common-prop-types';
 
 // Error Routes
 const Page404 = React.lazy(() => import(/* webpackChunkName: "404" */ '../Modules/Page404'));
 
 // Order matters
-const initRoutesConfig = () => {
+const initRoutesConfig = (): TRouteConfig[] => {
     return [
         {
             path: routes.dxtrade,
-            // eslint-disable-next-line react/display-name
-            component: props => <CFD {...props} platform='dxtrade' />,
+            component: (props: TCFDDashboardProps) => <CFD {...props} platform='dxtrade' />,
             getTitle: () => localize('Deriv X'),
             is_authenticated: false,
         },
         {
             path: routes.mt5,
-            // eslint-disable-next-line react/display-name
-            component: props => <CFD {...props} platform='mt5' />,
+            component: (props: TCFDDashboardProps) => <CFD {...props} platform='mt5' />,
```

Comment:
should we use the enum here from `@deriv/shared`
```suggestion
            component: (props: TCFDDashboardProps) => <CFD {...props} platform={CFD_PLATFORMS.MT5} />,
```